Flow.Launcher.Test/FuzzyMatcherTest.cs

```diff
@@ -129,11 +131,11 @@ public void GivenQueryString_WhenAppliedPrecisionFiltering_ThenShouldReturnGreat
 
         [TestCase(Chrome, Chrome, 157)]
         [TestCase(Chrome, LastIsChrome, 147)]
-        [TestCase(Chrome, HelpCureHopeRaiseOnMindEntityChrome, 25)]
+        [TestCase(Chrome, HelpCureHopeRaiseOnMindEntityChrome, 90)]
         [TestCase(Chrome, UninstallOrChangeProgramsOnYourComputer, 21)]
         [TestCase(Chrome, CandyCrushSagaFromKing, 0)]
-        [TestCase("sql", MicrosoftSqlServerManagementStudio, 110)]
-        [TestCase("sql  manag", MicrosoftSqlServerManagementStudio, 121)]//double spacing intended
+        [TestCase("sql", MicrosoftSqlServerManagementStudio, 90)]
```

Comment:
this score has dipped due to acronym penalty?

Reply:
yes, it match the middle "SQL" but without the M